### PR TITLE
Update README.MD

### PR DESCRIPTION
**Pull request by mdomasevicius**

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    $ mage daemon
+    ```
+    
+    or 
+    
+    ```console
+    $ go run mage.go daemon
+    ```
+
+5. Create a PR with similar to [this one](https://github.com/mysteriumnetwork/node/pull/2720)
```
Comment (ignasbernotas) [inline README.MD:31]: unnecessary "with"

File: README.MD [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Set appropriate release version (i.e. 0.1.6 at the time of writing)
+
+2. Generate a changelog (github_changelog_generator --user mysteriumnetwork --project dvpn-web -t $GITHUB_TOKEN --since-tag 0.1.5)  
+Use generate changelog for release notes.
+
+3. Wait until builds pass:  
+    - https://gitlab.com/mysteriumnetwork/dvpn-web/-/pipelines
+    - https://gitlab.com/mysteriumnetwork/go-dvpn-web/-/pipelines
+
+4. On the node side you need to update dependency
```
Comment (ignasbernotas) [inline README.MD:15]: On the node side you will need to update the `go-dvpn-web`bundler dependency.

**Review by ignasbernotas — APPROVED** [review]

File: README.MD (hunk 1 above)
Comment (mdomasevicius) [inline README.MD:31]:
```suggestion
5. Create a PR similar to [this one](https://github.com/mysteriumnetwork/node/pull/2720) 
```

File: README.MD (hunk 2 above)
Comment (mdomasevicius) [inline README.MD:15]:
```suggestion
4. On the node side you will need to update the `go-dvpn-web` bundler dependency. 
```

**Review by Waldz — APPROVED** [review]

**Pull request merged**